trailblazer/clients/slurm_api_client.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+from trailblazer.clients.cancel_job_response import SlurmCancelJobResponse
+from trailblazer.clients.job_info_response import SlurmJobInfoResponse, SlurmJobsInfoResponse
+
+
+class SlurmRestApiClient:
+    def __init__(self, base_url: str, access_token: str, user_name: str) -> None:
+        self.base_url = base_url
+        self.headers = {"X-SLURM-USER-NAME": user_name, "X-SLURM-USER-TOKEN": access_token}
```

Comment:
Do we want to provide the tokens when we instantiate? I guess they can run out while using the client and then it might be good for the client to be able to set these by itself?

Reply:
The auth part of this will likely need some work. It might be acceptable to use a token which does not expire.